confd/templates/opennms.properties.d/collectd.properties.tpl

```diff
@@ -0,0 +1,12 @@
+#
+# DO NOT CHANGE THIS FILE! GENERATED FROM confd
+#
+org.opennms.threshd.value.decimalformat={{getv "/threshd/value-decimal-format" "##.##"}}
+{{$base := "/collectd" -}}
+org.opennms.collectd.instrumentationClass={{getv (print $base "/instrumentation-class") "org.opennms.netmgt.collectd.DefaultCollectdInstrumentation"}}
```

Comment:
Any reason not to use camelCase in our key names rather than hyphenating them? It would be nice if key names more closely matched the property they are tied to.

This applies to the rest of the keys as well.

Reply:
I missed that one.

During all the ideas that came to my mind due to the inconsistency with the OpenNMS property names (camel case in some cases, '-' in other cases, '_' in other cases, different parents for properties that belong to the same feature, etc.), I decided to use what OpenNMS has when possible and when it make sense, and during those multiple reworks, I missed some of them. Thank for catching it.I'll fix it, and try to find other cases.

All the examples from the example directory are working, so you can see how the YAML look like. Of course, to use the examples, you should build the container and then tag it to be consistent with the docker-compose YAML